USFMToolsSharp.Renderers.HTML/HTMLConfig.cs

```diff
@@ -20,6 +20,7 @@ public HTMLConfig()
             separateVerses = false;
             blankColumn = false;
             partialHTML = false;
+            hasTOC = false;
```

Comment:
This should be something along the lines of renderTableOfContents or displayTableOfContents, etc.

Reply:
Can you tell more about it? Do you mean it should be a method available to the client code or it should name differently?